skins/cat17/src/sass/layouts/pages/contact.scss

```diff
@@ -33,13 +33,51 @@
 			}
 		}
 		.wrap-submit {
-			margin-top: 70px;
 			width: 90%;
 			display: flex;
 			justify-content: center;
+			@include bkp(sm) {
+				margin-top: 70px;
+			}
 		}
 		.list-menu {
 			margin-top: 30px;
 		}
 	}
+	@include bkp(smmax) {
+		.col-sm-6 {
+			width: 100%;
```

Comment:
Ok, this works, but does't that go against the spirit of the bootstrap grid? I would have thought that the change would involve adding more column classes to the markup and let Bootstrap do its thing.

Reply:
Thanks a lot, I really was not familiar with that!